Automattic/jetpack
# Add health check field to post fields

This is a PR taken out from #25803. In this change there are the changes that involves the PHP changes to add the post field.

#### Other information:


#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






* See testing instructions on other PR
* This PR should not change any functionality yet, but publicize connections should work seamlessly from the editor and simple editor, and even on simple sites.
